sdks/python/apache_beam/runners/worker/bundle_processor.py

```diff
@@ -947,11 +955,11 @@ def process_bundle(self, instruction_id):
       # (transform_id, timer_family_id).
       data_channels = collections.defaultdict(
           list
-      )  # type: DefaultDict[data_plane.GrpcClientDataChannel, List[str]]
+      )  # type: DefaultDict[data_plane.GrpcClientDataChannel, List[Union[str, Tuple[str, str]]]]
```

Comment:
This can in fact contain items which are `Tuple[str, str]`.  see:

```python
        data_channels[self.timer_data_channel].extend(
            list(self.timers_info.keys()))
```

Reply:
Yep, this changed a couple of months ago. It'll be good to finally have these type annotations checked.